.bingo/Variables.mk

```diff
@@ -7,19 +7,37 @@ GO     ?= $(shell which go)
 
 # Below generated variables ensure that every time a tool under each variable is invoked, the correct version
 # will be used; reinstalling only if needed.
-# For example for golangci-lint variable:
+# For example for addlicense variable:
 #
 # In your main Makefile (for non array binaries):
 #
 #include .bingo/Variables.mk # Assuming -dir was set to .bingo .
 #
-#command: $(GOLANGCI_LINT)
-#	@echo "Running golangci-lint"
-#	@$(GOLANGCI_LINT) <flags/args..>
+#command: $(ADDLICENSE)
+#	@echo "Running addlicense"
+#	@$(ADDLICENSE) <flags/args..>
 #
+ADDLICENSE := $(GOBIN)/addlicense-v1.1.1
```

Comment:
Doesn't addlicense need the path for the license header to use? https://github.com/GoogleCloudPlatform/prometheus-engine/blob/main/hack/boilerplate.txt

Reply:
addlicense generates exactly our license by default.

Reply:
oh sweet